api/src/queries/project-queries.ts

```diff
@@ -16,11 +15,11 @@ const defaultLog = getLogger('queries/project-queries');
 /**
  * SQL query to insert a project row.
  *
- * @param {(PostProjectData & PostLocationData)} project
+ * @param {(PostProjectData & PostLocationData & PostCoordinatorData & PostObjectivesData)} project
  * @returns {SQLStatement} sql query object
  */
 export const postProjectSQL = (
-  project: PostProjectData & PostLocationData & PostCoordinatorData & PostObjectivesData & PostPermitData
+  project: PostProjectData & PostLocationData & PostCoordinatorData & PostObjectivesData
```

Comment:
awesome, noticed this when working on my ticket yesterday - nice to clean this up 🦖